app/views/tasks/employee_tasks.html.erb

```diff
@@ -41,17 +26,23 @@
       <td>
         <%= task.status %>
       </td>
+      <td>
+        <%= task.complexity %>
+      </td>
       <td>
         <%= task.expected_end_date.to_formatted_s(:short) %>
       </td>
       <td>
-        <%= form_for task, url: update_task_logtime_path(task), remote: true, html: { method: :patch } do |form|%>
+        <%= task_logtime(task) %>
+      </td>
+      <td>
+        <%= form_for task.task_time_logs.where(employee_id: current_employee.id).first, url: update_task_logtime_path(task), remote: true, html: { method: :patch } do |form|%>
```

Comment:
do this in the controller 
`task.task_time_logs.where(employee_id: current_employee.id).first`

Reply:
umm i have to do it for every task, can't just pass one param here